package.json

```diff
@@ -32,9 +32,10 @@
     "node": ">= 8.0.0"
   },
   "devDependencies": {
-    "code": "^5.2.0",
-    "lab": "^15.4.4",
     "pg": "^7.4.3",
-    "standard": "^11.0.1"
+    "standard": "^12.0.1"
+  },
+  "dependencies": {
+    "tap": "^14.2.2"
```

Comment:
this should be e devDependency.

Reply:
how could I have missed that 😣

fixed it